packages/fx-core/src/plugins/resource/bot/v2/mapping.ts

```diff
@@ -48,6 +49,13 @@ const triggerScenariosMap: Map<string, string[]> = new Map<string, string[]>([
   [AppServiceOptionItemForVS.id, [TemplateProjectsScenarios.NOTIFICATION_WEBAPI_SCENARIO_NAME]],
 ]);
 
+const PlatformRuntimeMap: Map<Platform, Runtime> = new Map<Platform, Runtime>([
+  [Platform.VS, Runtime.Dotnet],
+  [Platform.VSCode, Runtime.Node],
+  [Platform.CLI, Runtime.Node],
```

Comment:
Does it mean `Platform.CLI` does support `Runtime.Dotnet`?

Reply:
There is a new feature to create dotnet projects in CLI being developed and it's controlled by isCliDontnetEnabled() flag. But currently CLI only supports Node projects.